src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -104,6 +104,11 @@ The choice of hardware per node determines the [cost](#step-2-select-the-cloud-p
     ----------|------------
     Transactions per second | Each vCPU can handle around 1000 transactions per second. For example, 2 vCPUs can handle 2000 transactions per second and 4 vCPUs can handle 4000 transactions per second.
     Scaling | When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For most production applications, we recommend **at least 4 to 8 vCPUs per node**.
+
+    {{site.data.alerts.callout_info}}
+    Clusters deployed in CockroachDB Cloud can be created with a minimum of 2vCPUs per node on AWS and GCP or 4vcPUs per node on Azure (Limited Access).
```

Comment:
Azure isn't in limited access; it's been GA-ed. Is that what's meant here and below by "Limited Access"?